modules/ROOT/pages/db/database-connector-examples.adoc

```diff
@@ -250,6 +250,14 @@ casting, which requires two colons before the type you are casting to, for examp
 `<db:sql>SELECT price\:\:float8 FROM PRODUCT</db:sql>`
 ====
 
+[NOTE]
+====
+*Since version 1.8.0 (SQL Casting for PostgreSQL and Snowflake)*:
+Database Connector now accepts SQL Casting PostgreSQL and Snowflake syntax's, through the double colon (::) expression without the need to escape each colon (this feature doesn't change the existing behavior for the connector). For example:
```

Comment:
`(this feature doesn't change the existing behavior for the connector)` which behaviour? character escaping? specify that

Reply:
yes, the escaping. fixed